src/main/resources/assets/jei/lang/de_DE.lang

```diff
@@ -1,64 +1,88 @@
 # Tooltips
-jei.tooltip.config=Konfiguration
+jei.tooltip.config=JEI-Konfiguration
+jei.tooltip.show.recipes=Zeige Rezepte
 jei.tooltip.show.all.recipes=Zeige alle Rezepte
 jei.tooltip.delete.item=Klicke zum Löschen
 jei.tooltip.liquid.amount.with.capacity=%,d / %,d mB
 jei.tooltip.liquid.amount=%,d mB
+jei.tooltip.transfer=Bewege Gegenstände
+jei.tooltip.recipe.ore.dict=Akzeptiert: %s
+jei.tooltip.item.colors=Farben: %s
+jei.tooltip.shapeless.recipe=Formloses Rezept
+jei.tooltip.cheat.mode=Cheat-Modus
 
 # Error Tooltips
-jei.tooltip.error.recipe.transfer.missing=Fehlende Items
-jei.tooltip.error.recipe.transfer.inventory.full=Inventar ist voll
+jei.tooltip.error.recipe.transfer.missing=Fehlende Gegenstände
```

Comment:
Ok, you are absoluty right in that example and I agree with that. But when the context and tooltips tell you that an "Item" and a "Gegenstand" is the same, there can no confusion happen, as it's the opposite case: It tells you what terms can be used synonyms, as they both mean the same.

Example:
config.jei.interface.overlayEnabled=Aktiviere Item-Liste
config.jei.interface.overlayEnabled.comment=Zeige die Liste der Gegenstände neben einem geöffneten Benutzer-Interface.

This tells everybody that a "Item-Liste" is a "Liste der Gegenstände", so a confusion is impossible, and the reader learned this by the way if he did not know before. Your argument that a confusion can happen when one of the terms is used with another meaning doesn't fit, as in the case of "Item" and "Gegenstand", there is equality in the information, they are used for the same thing. I know no game or other context where an "Item" occurs but is not a "Gegenstand", and I can't imagine any context where this could happen.